outport/drivers/factory/elasticDriverFactory.go

```diff
@@ -118,36 +112,33 @@ func createElasticProcessor(args *ArgsIndexerFactory) (indexer.ElasticProcessor,
 		ShardCoordinator:         args.ShardCoordinator,
 	}
 
-	return indexer.NewElasticProcessor(esIndexerArgs)
+	return elastic.NewElasticProcessor(esIndexerArgs)
 }
 
-func checkDataIndexerParams(arguments *ArgsIndexerFactory) error {
+func checkDataIndexerParams(arguments *ArgsElasticDriverFactory) error {
 	if arguments.IndexerCacheSize < 0 {
```

Comment:
Please add a nil check for arguments as we can pass nil and the code will break

Reply:
done.